contrib/cirrus/runner.sh

```diff
@@ -126,6 +126,12 @@ function _run_minikube() {
     showrun bats test/minikube |& logformatter
 }
 
+function _run_farm() {
+    _bail_if_test_can_be_skipped test/farm
```

Comment:
This should also include `test/system`:
```bash
    _bail_if_... test/farm test/system
```
Reason: the `load ../system/helpers.bash` line. So if that file changes, we need to confirm that those changes work here, too.

Not worth a repush just for this.

Reply:
Fixed